code/part-two/processor/index.js

```diff
@@ -5,7 +5,7 @@ const { InvalidTransaction } = require('sawtooth-sdk/processor/exceptions');
 const MojiHandler = require('./handler');
 
 
-// The default validator URL, not valid in a docker-compose environment
+// Set validator URL, the default is not valid in a docker-compose environment
```

Comment:
**Add word:** Set validator URL ~,~ `as` the default is not valid in a docker-compose environment